### PR TITLE
Set a frame around the face and eyes with color

### PR DESCRIPTION
**Pull request by Ifeanyijasper**
Set a frame of bright colour around the face and eyes for visibilites

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -63,10 +63,9 @@ def main():
 
         cv2.imshow('frame', frame)
 
-        if cv2.waitKey(1) & 0xFF == ord('q'):
-            break
-    cap.release()
```
Comment (xurror) [inline src/main.py:68]: Why did you remove this??
Isn't it necessary to stop the video capture?

File: src/test.py [hunk 2]
```diff
@@ -21,18 +21,18 @@ def test_with_image(img_path):
 
     if left_eye_pred == 0 and right_eye_pred == 0:
         score += 1
-        cv2.putText(img, "Closed", (10, height-20), font, 1, (255, 255, 255), 1, cv2.LINE_AA)
+        cv2.putText(img, "Asleep", (10, height-20), font, 1, (255, 255, 255), 1, cv2.LINE_AA)
```
Comment (xurror) [inline src/test.py:24]: This was already resovled in PR, did you pull before sending this??

File: src/test.py [hunk 3]
```diff
@@ -46,10 +46,9 @@ def test_with_image(img_path):
         cv2.readOpticalFlow(img, (0, 0), (width, height), (0, 0, 255), thicc)
 
     cv2.imshow('frame', img)
-
-    cv2.waitKey(0)
-
-    cv2.destroyAllWindows()
+    k = cv2.waitKey(0)
+    if k == 27:
+    	cv2.destroyAllWindows()
```
Comment (xurror) [inline src/test.py:51]: I see some indentation issues here. Did this work well?

**Review by xurror — CHANGES_REQUESTED** [review]

File: src/test.py (hunk 2 above)
Comment (Ifeanyijasper) [reply src/test.py:24]: It was done on the test .py not on main.py

File: src/main.py (hunk 1 above)
Comment (Ifeanyijasper) [reply src/main.py:68]: it is error message that is causing it to end

File: src/test.py (hunk 2 above)
Comment (Ifeanyijasper) [reply src/test.py:24]: Screenshot from 2020-05-14 11-22-10

File: src/test.py (hunk 3 above)
Comment (Ifeanyijasper) [reply src/test.py:51]: it is working on mine

File: src/test.py (hunk 2 above)
Comment (xurror) [reply src/test.py:24]: Thanks for clarifying that. I didn't notice it.
